docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,138 @@
+
+- Resolved technical currency by updating `socks` transitive dependency. ([#2049](https://github.com/zowe/zowe-cli/pull/2049))
+
+- Updated zos-files copy dsclp system tests to include large mock files. This improves system test coverage for edge cases involving large data sets. ([#2023](https://github.com/zowe/zowe-cli/pull/2023))
+
+#### Zowe CLI Imperative Framework
+
+- Fixed race condition in the `config convert-profiles` command that may fail to delete secure values for old profiles. ([Zowe CLI #2055](https://github.com/zowe/zowe-cli/pull/2055))
+
+- Resolved issue in `ProfileInfo` where schema comparisons fail, specifically when comparing the cached schema against a command-based schema during registration. ([Zowe CLI #2045](https://github.com/zowe/zowe-cli/pull/2045))
+
+- Resolved technical currency by updating `socks` transitive dependency. ([Zowe CLI #2049](https://github.com/zowe/zowe-cli/pull/2049))
+
+- Updated `mustache` and `jsonschema` dependencies for technical currency. ([Zowe CLI #2024](https://github.com/zowe/zowe-cli/pull/2024))
+
+- Fixed issue where secure property names could be returned for the wrong profile. ([Zowe Explorer #2633](https://github.com/zowe/vscode-extension-for-zowe/issues/2633))
+
+- Fixed issue when a property is not found in `ProfileInfo.updateProperty({forceUpdate: true})` so extenders can set or update a property in a team configuration even if it is not defined in the schema without their application/extension failing. ([Zowe Explorer #2493](https://github.com/zowe/vscode-extension-for-zowe/issues/2493))
+
+#### IMS Plug-in for Zowe CLI
+
+- Added missing shrinkwrap to specify versions of dependencies installed with the plug-in. ([#65](https://github.com/zowe/zowe-cli-ims-plugin/pull/65))
+
+#### z/OS FTP Plug-in for Zowe CLI
+
+- Fixed an issue specific to Windows where uploading a data set with JCL content hangs, preventing further actions through the zFTP VSCode Extension. ([#2533](https://github.com/zowe/vscode-extension-for-zowe/issues/2533))
+
+### Zowe Explorer
+
+#### Zowe Explorer (Core)
+
+- See the [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer API
+
+- See the [Zowe Explorer API](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-api/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer FTP Extension
+
+- See the [Zowe Explorer FTP Extension](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-ftp-extension/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer ESLint Plug-in
+
+- See the [Zowe Explorer ESLint Plug-in](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/eslint-plugin-zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+### Vulnerabilities fixed
```

Comment:
@janan07 -- This content is missing. Wasn't sure if you were waiting to add this yourself later on, but wanted to point this out just in case.